### ISSUE TITLE
[Intermediate] - Add generation page base on GQL API

### ISSUE THREAD

**Issue by igihcksn** [igihcksn/mypocketmonsters#31]
Base on this API https://github.com/mazipan/graphql-pokeapi we can show the generation to this project so need help to add new page to show list of generation

**Comment by rodolfonei** [comment]
Hi! I would like to work on this one, please

**Comment by igihcksn** [comment]
Yes sure @rodolfonei please follow existing component to build new page, and don't forget add the screenshot
